### PR TITLE
Fixed topic echo test

### PR DESCRIPTION
**Pull request by jennuine**
# 🦟 Bug fix

Part of https://github.com/gazebosim/gz-sim/issues/1575

## Summary
Fixed the `TopicEcho_TEST`.

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- ~~[ ] Updated documentation (as needed)~~
- ~~[ ] Updated migration guide (as needed)~~
- ~~[ ] Consider updating Python bindings (if the library has them)~~
- [x] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [x] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by chapulina — APPROVED** [review]
Nice! LGTM if it passes CI.

File: src/plugins/topic_echo/TopicEcho_TEST.cc [hunk 1]
```diff
@@ -16,224 +16,261 @@
...
 
 /////////////////////////////////////////////////
-TEST(TopicEchoTest, Load)
+TEST(TopicEchoTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(Load))
 {
-  EXPECT_TRUE(initApp());
+  common::Console::SetVerbosity(4);
+
+  Application app(g_argc, g_argv);
+  app.AddPluginPath(std::string(PROJECT_BINARY_PATH) + "/lib");
```
Comment (ahcorde) [inline src/plugins/topic_echo/TopicEcho_TEST.cc:54]: common::joinPaths

File: src/plugins/topic_echo/TopicEcho_TEST.cc [hunk 2]
```diff
@@ -16,224 +16,261 @@
...
 /////////////////////////////////////////////////
-TEST(TopicEchoTest, Echo)
+TEST(TopicEchoTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(Echo))
 {
-  setVerbosity(4);
-  EXPECT_TRUE(initApp());
+  common::Console::SetVerbosity(4);
+
+  Application app(g_argc, g_argv);
+  app.AddPluginPath(std::string(PROJECT_BINARY_PATH) + "/lib");
```
Comment (ahcorde) [inline src/plugins/topic_echo/TopicEcho_TEST.cc:80]: common::JoinPaths

**Review by ahcorde — CHANGES_REQUESTED** [review]

File: src/plugins/topic_echo/TopicEcho_TEST.cc (hunk 2 above)
Comment (jennuine) [reply src/plugins/topic_echo/TopicEcho_TEST.cc]: 118d4d5943ecf97a498ae4facd9a7091fb6e417b

File: src/plugins/topic_echo/TopicEcho_TEST.cc (hunk 1 above)
Comment (jennuine) [reply src/plugins/topic_echo/TopicEcho_TEST.cc]: 118d4d5943ecf97a498ae4facd9a7091fb6e417b

**Review by chapulina — APPROVED** [review]
Thanks for bringing that test back!

**Pull request merged**